src/util/keyboardSwitch.js

```diff
@@ -0,0 +1,49 @@
+const keyboardSwitch = (e, octave) => {
```

Comment:
I like that this is still set up to be dynamic. Maybe we'll revisit this but not necessary with the new features you've added.